lib/rabbitmq/cli/command_behaviour.ex

```diff
@@ -29,7 +29,9 @@ defmodule RabbitMQ.CLI.CommandBehaviour do
                       scopes: 0,
                       usage_additional: 0,
                       switches: 0,
-                      aliases: 0
+                      aliases: 0,
+                      required_rabbit_app_state: 2,
+                      validate_execution_environment: 2
```

Comment:
Every command either needs RabbitMQ running or stopped. It's not something optional. Under what circumstances will a command require a running app when `--offline` is passed?

I mentioned this in https://github.com/rabbitmq/rabbitmq-cli/pull/215#issuecomment-320220048.